app/src/main/java/org/systers/mentorship/view/adapters/TasksAdapter.kt

```diff
@@ -7,15 +7,17 @@ import android.view.ViewGroup
 import kotlinx.android.synthetic.main.task_list_item.view.*
 import org.systers.mentorship.MentorshipApplication
 import org.systers.mentorship.R
+import org.systers.mentorship.models.Task
 
 /**
  * This class represents the adapter that fills in each view of the Tasks recyclerView
- * @param taskstsList list of tasks taken up by the mentee
+ * @param tasksList list of tasks taken up by the mentee
```

Comment:
thanks for fixing this typo ;)

Reply:
Thanks :) I also found quite a few warnings that can be easily resolved in different files I worked on. Can I perhaps create a code quality issue to resolve them?